### ISSUE TITLE
Can't enable hot reloading

### ISSUE THREAD

**Issue by kianfucius** [facebook/react-native#19682]
Hi friends, I currently learning how to develop my first app with react-native.
I followed the installation guide at: https://facebook.github.io/react-native/docs/getting-started.html

I am coding in Android Studio on Windows.
Unfortunately, I cannot figure out how to enable hot reloading and nothing online seems to be working.
I'm really at a loss for what to do here.
From what I understand, I should be able to do yarn start from within my project folder; if I make any sort of minor changes and save in Android Studio, the changes should show up on the app within a short amount of time. However, when I make changes to app.js and save, nothing happens.

I have the app running in developer mode (as opposed to production), and have tried to shake the device using the virtual sensors or use CTRL-M, to no avail (the purpose for this is to open the developer menu). I have send keyboard shortcuts set to Emulator Controls (default). SOS this is driving me insane because I feel like the fix is easy but I can't figure it out.

TL;DR working with yarn, android studio, windows dev environment; can't figure out how to open developer menu.

Thanks,
kianfucius
